roles/moving_objects_between_buckets/README.md

```diff
@@ -0,0 +1,87 @@
+moving_objects_between_buckets
+==================
+
+A role to move objects from one S3 Bucket to another.
```

Comment:
Can you please update description and role variables if the role has new functionalities?